<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WAttention</class>
 <widget class="QWidget" name="WAttention">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="lyTop" stretch="0,1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btExt">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(80, 80, 80);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbTop">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Сообщение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCls">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lySpl" stretch="0,1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frTLS">
       <property name="minimumSize">
        <size>
         <width>4</width>
         <height>4</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>4</width>
         <height>4</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>SizeFDiagCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frTop">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>4</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>4</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>SizeVerCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frTRS">
       <property name="minimumSize">
        <size>
         <width>4</width>
         <height>4</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>4</width>
         <height>4</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>SizeBDiagCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyMid" stretch="0,0,0">
     <item>
      <widget class="QFrame" name="frLft">
       <property name="minimumSize">
        <size>
         <width>4</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>4</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spc">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frRgt">
       <property name="minimumSize">
        <size>
         <width>4</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>4</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyBtm" stretch="0,1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frBLS">
       <property name="minimumSize">
        <size>
         <width>4</width>
         <height>4</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>4</width>
         <height>4</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>SizeBDiagCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frBtm">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>4</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>4</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>SizeVerCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frBRS">
       <property name="minimumSize">
        <size>
         <width>4</width>
         <height>4</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>4</width>
         <height>4</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>SizeFDiagCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
